frontend/apps/reader/modules/readerfont.lua

```diff
@@ -50,6 +51,24 @@ function ReaderFont:init()
     table.insert(self.face_table, {
         text = _("Font settings"),
         sub_item_table = self:getFontSettingsTable(),
+    })
+    table.insert(self.face_table, {
+        text_func = function()
+            local nb_family_fonts = 0
+            local g_font_family_fonts = G_reader_settings:readSetting("cre_font_family_fonts", {})
+            for family, name in pairs(g_font_family_fonts) do
+                if self.font_family_fonts[family] then
+                    nb_family_fonts = nb_family_fonts + 1
+                elseif g_font_family_fonts[family] and self.font_family_fonts[family] ~= false then
+                    nb_family_fonts = nb_family_fonts + 1
+                end
+            end
+            if nb_family_fonts > 0 then
+                return T(_("Font-family fonts (%1)"), nb_family_fonts)
+            end
+            return _("Font-family fonts")
```

Comment:
Might need a translators comment to explain.

Reply:
Any suggestion ?
`font-family is a CSS property name, keep it untranslated` ?

A bigger explanation, better that we can do in one line, is in the help_text they will probably meet not far from there.

Reply:
Yes, I think that's good